pkg/helm/handlers/handlers.go

```diff
@@ -146,6 +150,39 @@ func (h *helmHandlers) HandleHelmInstall(user *auth.User, w http.ResponseWriter,
 	w.Write(res)
 }
 
+func (h *helmHandlers) HandleHelmInstallAsync(user *auth.User, w http.ResponseWriter, r *http.Request) {
```

Comment:
There is a lot of duplicate code between this and the synch request, can you refactor to bring common code into separate method. Same for upgrade.

Reply:
The reason the code is duplicated because we would be removing the sync install/upgrade api after ui transition to using async api's.

Reply:
OK makes sense, yeah I agree is not worth spending time on refactoring for code sharing.